src/Object/AssignDeep.ts

```diff
@@ -0,0 +1,39 @@
+import {Iteration} from '../Iteration/Iteration'
```

Comment:
Why create a whole file for `AssignDeep` if we already coded it elsewhere? Why not re-export `Assign` with the `'deep'` option ticked in already? I'm against exporting a `Deep` utility for `Assign`, just pass `'deep'`.